code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -66,7 +66,7 @@
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
 	// this happens before the parent call because `del_on_death` may be set
-	if(can_die() && !admin_spawned)
```

Comment:
I wasn't sure why this was a thing, and it only made it harder to test things. If anyone can tell me why then I'll happily put it back in.

Reply:
Looking at the code, I'm assuming that folks do not want to award a player with a `medal_type` or `score_type` if the megafauna the player killed is spawned in by an Admin.

Reply:
the medal system is disabled on para because people would flaunt it, "i killed 200 drakes more than you so i am a better miner", so that doesn't really matter.